cloudinit/config/cloud-init-schema.json

```
@@ -1343,6 +1343,305 @@
           }
         }
       }
+    },
+    "cc_scripts_vendor": {
+      "type": "object",
+      "properties": {
+        "vendor_data": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "enabled": {
+              "type": ["boolean", "string"],
+              "description": "Whether vendor data is enabled or not. Use of string for this value is DEPRECATED. Default: ``true``"
+            },
+            "prefix": {
+              "type": ["boolean", "string"],
```

I think we added wrong "types" in the latest commit for prefix it should be a 
```suggestion
              "type": ["array", "string"],
              "items": {"type": ["string", "integer"]},
              "minItems": 1,
```

---

@TheRealFalcon CI is failing on this invalid schema definition I believe. Otherwise I think we are good to go.

---

Gah, sorry. I made the change but never pushed it.